corehq/util/test_utils.py

```diff
@@ -400,9 +400,9 @@ def create_and_save_a_form(domain):
 
 def _create_case(domain, **kwargs):
     from casexml.apps.case.mock import CaseBlock
-    from casexml.apps.case.util import post_case_blocks
-    return post_case_blocks(
```

Comment:
I don't think so. I think the reason the both exist is from before they were separate repo's and casexml doesn't care about form metadata which is probably why it was never included.